external-resolver/src/main/scala/org/genivi/sota/resolver/filters/FilterRepository.scala

```diff
@@ -32,10 +33,13 @@ object FilterRepository {
   def add(filter: Filter)(implicit ec: ExecutionContext): DBIO[Filter] =
     (filters += filter).map(_ => filter)
 
-  def load(name: Filter.Name)(implicit ec: ExecutionContext): DBIO[Option[Filter]] =
+  def exists(name: Filter.Name)(implicit ec: ExecutionContext): DBIO[Filter] =
```

Comment:
I don't like failing DB action. For me it has semantic of failing db request, and it is not the case here